src/libraries/System.Formats.Nrbf/tests/AttackTests.cs

```diff
@@ -108,11 +108,15 @@ public void CyclicClassReferencesInArraysOfTDoNotCauseStackOverflow()
     }
 
 #if !NETFRAMEWORK
+    // The tests need to ensure that 2GB+ does not get pre-allocated.
+    // 200k is enough to get the job done and avoid getting false positives.
+    const long AllocationThreshold = 200_000;
+
     // GC.GetAllocatedBytesForCurrentThread() is not available on Full Framework.
     // AppDomain.CurrentDomain.MonitoringTotalAllocatedMemorySize is available,
     // but it reports allocations for all threads. Using this API would require
     // ensuring that it's the only test that is being run at a time.
-    // Mono either allocates more than expected or the API is not precise enough
+    // Mono either allocates more than expected or the API is not precise enough.
```

Comment:
Is this comment still valid after this change?

Reply:
I am going to use the CI to find out